sdk/apimanagement/Azure.ResourceManager.ApiManagement/src/Customized/ApiManagementUserResource.cs

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#nullable disable
+
+using System;
+using System.Globalization;
+using Azure.Core;
+
+namespace Azure.ResourceManager.ApiManagement
+{
+    public partial class ApiManagementUserResource
+    {
+        private static readonly ResourceType ResourceTypeWithGroup = "Microsoft.ApiManagement/service/groups/users";
+
+        internal static void ValidateResourceId(ResourceIdentifier id)
```

Comment:
Use this kind of customized method to make sure the verification succeeds when calling the method [CreateGroupUserAsync](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/apimanagement/Azure.ResourceManager.ApiManagement/src/Generated/ApiManagementGroupResource.cs#L412)